src/components/bookmarks/AddListModal/AddListMoadalContents.tsx

```diff
@@ -0,0 +1,49 @@
+import { ChangeEvent, useState } from 'react';
+import {
+  addListCompleteButton,
+  addListModalInput,
+  contentsInputContainer,
+  maxLengthText,
+} from './AddListModal.css';
+
+interface AddListModalContentsProps {
+  listName: string;
+  onChangeListName: (name: string) => void;
+  addBookmarkList: () => void;
+}
+
+export default function AddListModalContents({
+  listName,
+  onChangeListName,
+  addBookmarkList,
+}: AddListModalContentsProps) {
+  const listNameLength = listName.length;
+
+  const onChangeModalContents = (e: ChangeEvent<HTMLInputElement>) => {
+    onChangeListName(e.target.value);
+  };
+  const hasContents = (contentsLength: number) => {
+    if (contentsLength > 0) return true;
+    return false;
+  };
```

Comment:
```suggestion
  const hasContents = (contentsLength: number) => {
    return contentsLength > 0
  };
```
이렇게 표현할 수 있을 것 같고, 개인적으로 이렇게 간단한 구문은 함수 없이 바로 사용해도 괜찮다는 의견입니다 ㅎㅎ